icepyx/core/is2ref.py

```diff
@@ -423,30 +425,33 @@ def extract_version(filepath, auth=None):
         f = h5py.File(filepath, "r")
 
     # Read the version information
+    version_str: str
     try:
-        version = f["METADATA"]["DatasetIdentification"].attrs["VersionID"]
+        version = f["METADATA"]["DatasetIdentification"].attrs["VersionID"]  # pyright: ignore[reportIndexIssue]
```

Comment:
`h5py` isn't annotated to allow accessing metadata with subscript notation. Just telling the typechecker "it's fine" for now :)